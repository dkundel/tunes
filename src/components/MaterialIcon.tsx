import * as React from 'react';

export interface MaterialIconProps {
  icon: string;
}

export default class PlayerButton extends React.Component<
  MaterialIconProps,
  undefined
> {
  public render() {
    return (
      <i className="material-icons">
        {this.props.icon}
      </i>
    );
  }
}
