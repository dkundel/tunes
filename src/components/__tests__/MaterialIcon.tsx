import * as React from 'react';
import MaterialIcon from '../MaterialIcon';
import * as renderer from 'react-test-renderer';

describe('test MaterialIcon component', () => {
  test('verify snapshot', () => {
    const component = renderer.create(<MaterialIcon icon="close" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
