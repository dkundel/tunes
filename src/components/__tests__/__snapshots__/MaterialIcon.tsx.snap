// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test MaterialIcon component verify snapshot 1`] = `
<i
  className="material-icons"
>
  close
</i>
`;
